### PR TITLE
Remove pkg's imported twice

### PR DESCRIPTION
**Pull request by testwill**

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
Thanks @testwill for your contribution! 

a couple of nits before moving forward
- Could you sign your commit (https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) 
- Could you split it into 2 commits one for each file and have the commit title be the file name, example
`nfd-master-internal_test.go rm pkg imported twice` 

/retitle Remove pkg's imported twice

File: test/e2e/node_feature_discovery_test.go [hunk 1]
```diff
@@ -35,7 +35,6 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	clientset "k8s.io/client-go/kubernetes"
 	taintutils "k8s.io/kubernetes/pkg/util/taints"
-	"k8s.io/kubernetes/test/e2e/framework"
 	e2elog "k8s.io/kubernetes/test/e2e/framework"
```
Comment (marquiz) [inline test/e2e/node_feature_discovery_test.go:38]: Drop the `e2elog` instead

```suggestion
	"k8s.io/kubernetes/test/e2e/framework"
```

and change all `e2elog.Logf` to `framework.Logf`

Comment (ArangoGutierrez) [reply test/e2e/node_feature_discovery_test.go:38]: @testwill , once this changes are in, I think we can move forward with this contribution.

thanks for your contribution to NFD!!

**Review by ArangoGutierrez — APPROVED** [review]
Thanks for the contribution !!!

**Review by marquiz — COMMENTED** [review]
Thankx @testwill
/lgtm

**Pull request merged**